otter/integration/tests/test_pause.py

```diff
@@ -91,6 +92,54 @@ def test_pause_stops_convergence(self):
                     server_build_time + convergence_interval * 2)
         # The group still thinks that server is building
         yield self.helper.assert_group_state(group, one_building)
+        returnValue(group)
+
+    @skip_me("Until resume is implemented: #1605")
+    @skip_if(not_mimic, "This requires mimic for server build time")
+    @inlineCallbacks
+    def test_resume(self):
+        """
+        Calling resume on a paused group changes {"paused": False} in group
+        state and triggers convergence. The group can then execute policy,
+        webhook or trigger convergence
+        """
+        group = yield self.test_pause_stops_convergence()
+        yield group.resume()
+        yield self.helper.assert_group_state(
+            group, ContainsDict({"paused": Equals(False)}))
+        yield group.wait_for_state(
+            self.rcs,
+            ContainsDict({"activeCapacity": Equals(1),
+                          "pendingCapacity": Equals(0),
+                          "desiredCapacity": Equals(1),
+                          "status": Equals("ACTIVE")}))
+
+        # can create and execute policy
+        policy = ScalingPolicy(set_to=1, scaling_group=group)
```

Comment:
`set_to` sets the desired capacity to 1.  I think we probably want `scale_by` here?